demos/object_detection_demo/python/object_detection_demo.py

```diff
@@ -77,6 +77,11 @@ def build_argparser():
     common_model_args.add_argument('--layout', type=str, default=None,
                                    help='Optional. Model inputs layouts. '
                                         'Ex. NCHW or input0:NCHW,input1:NC in case of more than one input.')
+    common_model_args.add_argument('--is_legacy', default=False, action='store_true',
```

Comment:
not sure that it is good idea to call parameter is_legacy in demo, maybe better use different arch types with different value inside realization?

Reply:
Yeah, the same approach is used in human_pose_estimation_demo: there are two arch types (ae, higherhrnet) for one `HpeAssociativeEmbedding` wrapper

Reply:
Done